app/src/main/java/org/simple/clinic/user/ForgotPinResult.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.user
+
+sealed class ForgotPinResult {
+  object Success : ForgotPinResult()
+  object NetworkError : ForgotPinResult()
+  object UserNotFound : ForgotPinResult()
+  data class UnexpectedError(val cause: Throwable? = null) : ForgotPinResult()
```

Comment:
Will the cause ever be null?

Reply:
Maybe. Sometimes we may just want to ignore the cause and not care about it.